src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -118,6 +124,12 @@ public function __construct(ClassMetadataFactoryInterface $classMetadataFactory
         }
         $this->classDiscriminatorResolver = $classDiscriminatorResolver;
         $this->objectClassResolver = $objectClassResolver;
+
+        $this->propertyAccessor = $propertyAccessor ??
+            PropertyAccess::createPropertyAccessorBuilder()
```

Comment:
the CS is a bit ugly. We usually don't do that in symfony
use:
```php
$this->propertyAccessor = $propertyAccessor ?? PropertyAccess::createPropertyAccessorBuilder()
   ->...
```

BTW, does it make sens to add `ropertyAccess::createDefaultPropertyAccessorBuilder()` with everything "well" setup?

Reply:
This is not the "default" setup since we disable errors when paths are not found & disable static calls (which are both enabled in usual PropertyAccessor)

Reply:
ok